ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutorFactory.java

```diff
@@ -136,8 +131,11 @@ static StandaloneExecutor create(
         udfLoader,
         true,
         versionChecker,
-        schemaInjectorFactory,
-        DefaultTopicInjector::new
+        (ec, sc) -> InjectorChain.of(
+            new DefaultSchemaInjector(
```

Comment:
It doesn't matter in what order the schema injector and topic injector are applied, right? The code and tests seem to be inconsistent about it.

Reply:
it should not matter at the current revision, but Andy wanted a test for it so there's a test for it. I guess it might help in case some one assumes something in the future. I will comb through and check for any inconsistencies since clearly the tests aren't doing their job... which ones were you talking about?

Reply:
I think the code was consistent about applying the schema injector before the topic injector (check me on this though -- didn't look that closely), but some of the tests had it the other way around, for example line 337 of RequestValidatorTest.

Reply:
There was indeed one time when it was flipped :P fixed!